_includes/en/Scripts.md

```diff
@@ -0,0 +1,29 @@
+Daml script is a scripting language to run Daml commands against a ledger. For example:
+
+```
+module Test where
+
+import Daml.Script
+
+test = Script ()
+test = do 
+  alice <- allocateParty "Alice" 
+  bob <- allocateParty "Bob"
+  c <- submit alice $ createCmd NewContract with ...
+  submit bob $ exerciseCmd c Accept with ...
+```
+
+Scripts are compiled like usual Daml code to a `dar` package with the `daml build` command.
+
+| -------------------- | ----------------------------------------------------- |
+| Running a script | `daml script --dar example-0.0.1.dar --script-name ScriptExample:test --ledger-host localhost --ledger-port 6865`
+| Running a script with initial arguments given | `daml script --dar example-0.0.1.dar --input-file arguments_in_damllf_json.json --script-name ScriptExample:test --ledger-host localhost --ledger-port 6865` |
```

Comment:
Also perhaps a link to a reference on how to produce `arguments_in_damllf_json.json`?

Reply:
That would just point to the daml scripts docs, that's already mentionened.